Dockerfile

```diff
@@ -22,16 +22,18 @@ LABEL "org.opencontainers.image.version"="$IMAGE_VERSION"
 
 # Install the arch specific build of SSM agent *and confirm that it installed* -
 # yum will allow architecture-mismatched packages to not install and consider
-# the run successful. 
+# the run successful.
 # SSM Agent is downloaded from eu-north-1 as this region gets new releases of SSM Agent first.
 COPY ./hashes/ssm ./hashes
 RUN \
   curl -L "https://s3.eu-north-1.amazonaws.com/amazon-ssm-eu-north-1/${SSM_AGENT_VERSION}/linux_${ARCH}/amazon-ssm-agent.rpm" \
        -o "amazon-ssm-agent-${SSM_AGENT_VERSION}.${ARCH}.rpm" && \
   grep "amazon-ssm-agent-${SSM_AGENT_VERSION}.${ARCH}.rpm" hashes | sha512sum --check - && \
-  yum install -y "amazon-ssm-agent-${SSM_AGENT_VERSION}.${ARCH}.rpm" shadow-utils && \
+  yum install -y "amazon-ssm-agent-${SSM_AGENT_VERSION}.${ARCH}.rpm" shadow-utils jq && \
   rm "amazon-ssm-agent-${SSM_AGENT_VERSION}.${ARCH}.rpm" && \
-  rm -rf /var/cache/yum ./hashes
+  rm -rf /var/cache/yum ./hashes && \
+  rmdir /var/lib/amazon/ssm && \
+  ln -snf /.bottlerocket/host-containers/control/ssm /var/lib/amazon/ssm
```

Comment:
Maybe explain that `/.bottlerocket/host-containers/control` will be mounted when it runs on Bottlerocket? Dunno, maybe that's obvious.

Is this the first time that the control container becomes sensitive to it having the exact name `control` in Bottlerocket. i.e. before this I could have used this container and called it host-containers.foo on Bottlerocket and it would have worked? Does a host container have a way of finding out its name at runtime, and if so maybe we could create the link at runtime?

I'm kind of surprised that we mount host container paths in this non-abstracted way to begin with. I would have thought we would mount `/.bottlerocket/host-containers/control/` as `bottlerocket` for a host container or something like that to decouple things, but I guess I wasn't paying attention to that.

Reply:
Yeah, making ourselves sensitive to host container name would be unfortunate.  Do we have another way to get this name, or should we change the mounting strategy, or create some other way to use a consistent path instead of having to know the name..?

Reply:
@tjkirch might have more background here, but my guess is that if someone is using a custom name for their control container, they are probably sporting a custom Dockerfile as well. I can certainly add a comment though as a heads up though.

Reply:
What if you want "control" to be your custom container, and "control-br" to be the official one?  I'm sure there are other similar cases.  There's no need to tie ourselves to a given name such that it'd break when the user customizes something we let them customize.

Reply:
> ... Do we have another way to get this name, or should we change the mounting strategy, or create some other way to use a consistent path instead of having to know the name..?

This is where my mind went first.  Where are we going to get the name?  Off-the-cuff:  we could potentially pass the name into the container at runtime via the env file?  Just spitballing here...

Reply:
To address this, I've put out a PR on bottlerocket [#1416](https://github.com/bottlerocket-os/bottlerocket/pull/1416). By adding a second bind mount with a more generic name we can more safely reference the location in scripts.